config-windows.md

```diff
@@ -94,3 +94,42 @@ The following parameters can be specified:
         }
    }
 ```
+
+## <a name="configWindowsCredentialSpec" />Credential Spec
+
+You can configure a container's group Managed Service Accounts (gMSAs) via the OPTIONAL `credentialspec` field of the Windows configuration. For more information about gMSAs, see [Active Directory Service Accounts for Windows Containers][gMSAOverview]. For more information about tooling to generate a gMSA, see [Deployment Overview][gMSATooling]. The `credentialspec` MUST be a string containing an escaped JSON object.
```

Comment:
Yes, I can change this to be `interface{}`. (Done).  I'm not sure what this should be in `schema\config-windows.json` though if not type `string`? This is unique - no similar example exists in the spec to duplicate.

I have also updated the description in `config-windows.md` to indicate it's implementation-defined.

To be clear though - you are recommending to remove the example (even though technically accurate)? And what about the links? Should they stay (I think they should...)?